packages/next/src/lib/metadata/types/manifest-types.ts

```diff
@@ -1,16 +1,54 @@
+type ManifestCategory = 
+    | "books"
+    | "business"
+    | "education"
+    | "entertainment"
+    | "finance"
+    | "fitness"
+    | "food"
+    | "games"
+    | "government"
+    | "health"
+    | "kids"
+    | "lifestyle"
+    | "magazines"
+    | "medical"
+    | "music"
+    | "navigation"
+    | "news"
+    | "personalization"
+    | "photo"
+    | "politics"
+    | "productivity"
+    | "security"
+    | "shopping"
+    | "social"
+    | "sports"
+    | "travel"
+    | "utilities"
+    | "weather" 
+    
 export type Manifest = {
   background_color?: string
-  categories?: string[]
+  categories?: ManifestCategory[]
   description?: string
+  dir?: 'ltr' | 'rtl' | 'auto'
   display?: 'fullscreen' | 'standalone' | 'minimal-ui' | 'browser'
-  display_override?: string[]
+  display_override?: ('fullscreen' | 'standalone' | 'minimal-ui' | 'browser' | 'window-controls-overlay')[]
+  file_handlers?: {
+    action: string
+    accept: {
+      [mimeType: string]: string[]
+    }[]
+  }[]
   icons?: {
     src: string
     type?: string
     sizes?: string
     purpose?: 'any' | 'maskable' | 'monochrome' | 'badge'
   }[]
   id?: string
+  lang?: string
```

Comment:
there's not a `lang` property in the standard

Reply:
In the [Web Application Manifest](https://w3c.github.io/manifest/#lang-member) specification mentioned on the [MDN](https://developer.mozilla.org/en-US/docs/Web/Manifest) page, there is a `lang` property

I don't know why it wasn't added to the MDN documentation